packages/geo/src/bounds.ts

```diff
@@ -153,6 +168,30 @@ export class Bounds implements BoundingBox {
         return new Bounds(this.x - bounds.x, this.y - bounds.y, this.width, this.height);
     }
 
+    /**
+     * Find the bounds of a GeoJson MultiPolygon
+
+     * @param multipoly the polygon to measure
+     * @throws Error if `multipoly` is empty
+     */
+    public static fromMultiPolygon(multipoly: number[][][][]): Bounds {
+        if (multipoly.length == 0) return new Bounds(0, 0, 0, 0);
+        let minX = multipoly[0][0][0][0];
+        let minY = multipoly[0][0][0][1];
+        let maxX = minX;
+        let maxY = minY;
+        for (const [poly] of multipoly) {
```

Comment:
do we have any concerns about polygons wrapping the wrong way if the polygon crosses the antimeridaian?

Reply:
No because we are in source units not WSG84